doc/sphinx/changes.rst

```diff
@@ -2,6 +2,441 @@
+  corresponding definitions is additionally high (PR #8215).
+
+- A few improvements in inference of the return clause of `match` can
+  exceptionally introduce incompatibilities (PR #262). This can be
+  solved by writing an explicit `return` clause, sometimes even simply
+  an explicit `return _` clause.
+
+- Using non-projection values with the projection syntax is not
+  allowed. For instance "0.(S)" is not a valid way to write "S 0".
+  Projections from non-primitive (emulated) records are allowed with
+  warning "nonprimitive-projection-syntax".
+
+Kernel
+
+- Added primitive integers
+
+- Unfolding heuristic in termination checking made more complete.
+  In particular Coq is now more aggressive in unfolding constants
+  when it looks for a iota redex. Performance regression may occur
+  in Fixpoint declarations without an explicit {struct} annotation,
+  since guessing the decreasing argument can now be more expensive.
+  (PR #9602)
+
+Notations
+
+- New command `Declare Scope` to explicitly declare a scope name
+  before any use of it. Implicit declaration of a scope at the time of
+  `Bind Scope`, `Delimit Scope`, `Undelimit Scope`, or `Notation` is
+  deprecated.
+
+- New command `String Notation` to register string syntax for custom
+  inductive types.
+
+- Numeral notations now parse decimal constants such as 1.02e+01 or
+  10.2. Parsers added for Q and R. This should be considered as an
+  experimental feature currently.
+  Note: in -- the rare -- case when such numeral notations were used
+  in a development along with Q or R, they may have to be removed or
+  deconflicted through explicit scope annotations (1.23%Q,
+  45.6%R,...).
+
+- Various bugs have been fixed (e.g. PR #9214 on removing spurious
+  parentheses on abbreviations shortening a strict prefix of an application).
+
+- Numeral Notations now support inductive types in the input to
+  printing functions (e.g., numeral notations can be defined for terms
+  containing things like `@cons nat O O`), and parsing functions now
+  fully normalize terms including parameters of constructors (so that,
+  e.g., a numeral notation whose parsing function outputs a proof of
+  `Nat.gcd x y = 1` will no longer fail to parse due to containing the
+  constant `Nat.gcd` in the parameter-argument of `eq_refl`).  See
+  #9840 for more details.
+
+- Deprecated compatibility notations have actually been removed.  Uses
+  of these notations are generally easy to fix thanks to the hint
+  contained in the deprecation warnings. For projects that require
+  more than a handful of such fixes, there is [a
+  script](https://gist.github.com/JasonGross/9770653967de3679d131c59d42de6d17#file-replace-notations-py)
+  that will do it automatically, using the output of coqc. The script
+  contains documentation on its usage in a comment at the top.
+
+Plugins
+
+- The quote plugin (https://coq.inria.fr/distrib/V8.8.1/refman/proof-engine/detailed-tactic-examples.html#quote)
```

Comment:
This link could now be: https://coq.inria.fr/distrib/V8.9.0/refman/proof-engine/detailed-tactic-examples.html#quote

Reply:
Right